frontend/ui/uimanager.lua

```diff
@@ -34,10 +34,15 @@ local UIManager = {
 function UIManager:init()
     self.event_handlers = {
         __default__ = function(input_event)
-            self:sendEvent(input_event)
+            -- Suspension in Kobo can be interrupted by screen updates. We
+            -- ignore user touch input here so screen udpate won't be
+            -- triggered in suspend mode
+            if not Device.screen_saver_mode then
+                self:sendEvent(input_event)
+            end
```

Comment:
I didn't put this into the default handler for all models because it is a dangerous piece of code that can cause users to think their device is "bricked" if there is any bug in it. Are you also running into the same issue with kindle?